apps/server/src/modules/pseudonym/repo/external-tool-pseudonym.repo.ts

```diff
@@ -33,6 +33,14 @@ export class ExternalToolPseudonymRepo {
 		return domainObject;
 	}
 
+	async findPseudonymsByUserId(userId: EntityId): Promise<Pseudonym[]> {
+		const entities: ExternalToolPseudonymEntity[] = await this.em.find(ExternalToolPseudonymEntity, {
+			userId: new ObjectId(userId),
+		});
+		const pseudonyms: Pseudonym[] = entities.map((entity) => this.mapEntityToDomainObject(entity));
+		return pseudonyms;
```

Comment:
Could you please insert an empty line before the `return` call? It's not a change request, just a suggestion based on my preference - in my opinion it increases the code readability.